### PR TITLE
#162414155 install and setup coveralls

### PR DESCRIPTION
**Pull request by Luckzman**
#### What does this PR do?
setup coveralls integration for the project.

#### Description of Task to be completed?
The task entails installing and setting up coveralls for code coverage

#### How should this be manually tested?
The process is fully automated

#### What are the relevant pivotal tracker stories?
PT Link: https://www.pivotaltracker.com/story/show/162414155

#### Screenshots
screenshot 2019-01-09 at 3 48 46 pm

### CODE REVIEW

**Review by iMichaelOwolabi — DISMISSED** [review]
Good

**Review by eltNEG — DISMISSED** [review]
Ok

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
 node_modules
+<<<<<<< HEAD
```
Comment (codrex) [inline .gitignore]: Is there any reason why you have this here?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,5 @@
...
+=======
+.nyc_output
+>>>>>>> [ch-#162414155] install and setup coveralls
```
Comment (codrex) [inline .gitignore]: same as 👆

**Review by codrex — COMMENTED** [review]
Please attend to my comments

File: .gitignore (hunk 2 above)
Comment (Luckzman) [reply .gitignore]: fixed

File: .gitignore (hunk 1 above)
Comment (Luckzman) [reply .gitignore]: It's an oversight, I had issues with rebasing the initial conflicts. It's been fixed

**Review by ibraheemkabir — DISMISSED** [review]
ok.

**Review by iMichaelOwolabi — DISMISSED** [review]
:+1: Good job fixing all the issues @codrex pointed out

**Review by iAfolayan — DISMISSED** [review]

**Review by codrex — APPROVED** [review]

**Pull request merged**